### PR TITLE
Update TensorFlow, CUDA and OepnCV version in README

### PR DESCRIPTION
**Pull request by iizukak**
## Motivation and Context

Current Blueoil use tensorflow `1.13.1`, but some doc is still `1.4.1`. We should update some documentation.

## How has this been tested?

I did test inference execution with
- TensorFlow 1.13.1
- CUDA 10.0
- CuDNN 6.0
- Python3.6
- OpenCV 3.3.1

with Jetson Nano.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by iizukak** [comment]
I found some problem with README, I'll send other PR.

**Comment by iizukak** [comment]
run test

File: output_template/python/README.md [hunk 1]
```diff
@@ -87,15 +87,15 @@ $ python usb_camera_demo.py -c ../models/meta.yaml -m minimal_graph_with_shape.p
 
 ## Prerequisites
 ```
-Python = 3.5
-Opencv = 3.1.0
-python opencv = 3.1.0
+Python = 3.6
```
Comment (ruimashita) [inline output_template/python/README.md]: If the scripts work in python 3.5, `Python = 3.5, 3.6` is better.

File: output_template/python/README.md [hunk 2]
```diff
@@ -87,15 +87,15 @@ $ python usb_camera_demo.py -c ../models/meta.yaml -m minimal_graph_with_shape.p
...
+Opencv = 3.3.1
```
Comment (ruimashita) [inline output_template/python/README.md]: also here.
`Opencv => 3.1.0`

`python opencv => 3.1.0`

is better I think.

**Comment by iizukak** [comment]
@ruimashita Thanks. Fixed.

File: output_template/python/README.md [hunk 3]
```diff
@@ -87,15 +87,15 @@ $ python usb_camera_demo.py -c ../models/meta.yaml -m minimal_graph_with_shape.p
 
 ## Prerequisites
 ```
-Python = 3.5
-Opencv = 3.1.0
-python opencv = 3.1.0
+Python = 3.5, 3.6
+Opencv => 3.1.0
```
Comment (lm-jira) [inline output_template/python/README.md]: I've never seen `=>`.
Do you mean `>=`?

Comment (iizukak) [reply output_template/python/README.md]: OH, thanks.

Comment (iizukak) [reply output_template/python/README.md]: @lm-jira Fixed 🙇

**Pull request merged**